src/pkg/clouds/gcp/cloudrun.go

```diff
@@ -157,7 +157,7 @@ func (gcp Gcp) GetExecutionEnv(ctx context.Context, executionName string) (map[s
 
 	executionName = path.Base(executionName)
 	if len(executionName) < 6 {
-		return nil, fmt.Errorf("invalid execution name %q", executionName)
+		return nil, fmt.Errorf("invalid execution name: %q", executionName)
```

Comment:
Let's say why it's invalid:

```suggestion
		return nil, fmt.Errorf("invalid execution name, must be longer than 6 characters: %q", executionName)
```